graylog2-web-interface/src/components/widgets/WidgetFooter.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import WidgetFooter from 'components/widgets/WidgetFooter';
+
+describe('<WidgetFooter />', () => {
+  const date = new Date('December 17, 1995 03:24:00');
+  const lockWidget = true;
+  const disableReplay = true;
+
+  it('should render a widget footer locked and enabled replay', () => {
+    const wrapper = renderer.create(<WidgetFooter
+      locked={lockWidget}
```

Comment:
As `lockWidget` is always `true` in this scope, we can just write `locked` here.